docs/index.md

```diff
@@ -1,3 +1,11 @@
-# Shenanigan Wiki 🤸‍♀️️
+#  Introduction
 
-📣️ Join Shenanigan [discord](http://www.she.energy/join) 📣️
+Shenanigan is built by athlets for athtletes to bring a different perspective to your everyday life.
+
+Shenanigan provides a competitive and motivating platform for athletes of all types utilizing fiscal incentives to drive personal improvement. We are doing this because we believe that athletes should be payed for the hard work that the put in to train and hone their craft. 
```

Comment:
Put these two sentences at the beginning. Like this.

Shenanigan provides a competitive and motivating platform for athletes of all types utilizing fiscal incentives to drive personal improvement. 

We are doing this because we believe that athletes should be payed for the hard work that the put in to train and hone their craft. Shenanigan is built by athletes for athtletes to bring a different perspective to your everyday life.